pisces/settings.py

```diff
@@ -158,7 +158,8 @@
     "username": config.AS_USERNAME,
     "password": config.AS_PASSWORD,
     "repo": config.AS_REPO_ID,
-    "resource_id_0_prefixes": config.RESOURCE_ID_0_PREFIXES,
```

Comment:
Is the removal of "repo" here intentional?  I don't see it mentioned else where in the app, but if removed, shouldn't we also clean up `AS_REPO_ID` in config.py.example and config.py.deploy as well?

Reply:
I'm not seeing `repo` being removed here? If it is it's definitely unintentional, but I think the diff is referring to the change in handling `RESOURCE_ID_0_PREFIXES` which I'm now doing with `getattr`...

Reply:
Confirmed.  Looks fine now.  Wish I has a screenshot of the rendering from last week.